source/install.md

```diff
@@ -0,0 +1,245 @@
+$ mamba install -n <name> -c conda-forge <package(s)>
+```
+
+Or by adding the dependencies from an environment file:
+
+```shell-session
+$ mamba env update -n <name> -f <path to .yaml file>
+```
+
+Mamba/Conda cache the packages you install, so if you've already downloaded a particular version, you can install it in as many environments as you like without having to re-download it. In fact, Mamba/Conda try their hardest to unpack each version once, and then hardlink to it from each environment to keep the storage size of environments down. Be aware of this if you like to tinker with the code of your dependencies - your changes may affect other environments!
+
+The software in an environment can be automatically upgraded:
+
+```shell-session
+$ mamba upgrade -n <name> --all
+```
+
+But that can go wrong if you're mixing channels. Recreating the environment often works better:
+
+```shell-session
+$ mamba env remove -n <name>
+$ mamba create -n <name> -c conda-forge <package(s)>
+```
+
+Or specifying the particular package you want to update:
+
+```shell-session
+$ mamba upgrade -n <name> <package>
+```
+
+### Combining channels
+
+Sometimes, some of the software you want to use is not available on Conda Forge. In these cases, you can either install it in the usual way for your OS, or you can try combining channels:
+
+```shell-session
+$ mamba create -n psi4_and_bespokefit -c psi4 -c conda-forge -c anaconda openff-bespokefit psi4
+```
+
+In this case, this doesn't work because Conda/Mamba has to be very clever about how it solves this environment. It has to be able to choose versions of dependencies from the `anaconda` channel, even when those dependencies exist in the higher-priority `psi4` channel. It's generally recommended to use [strict channel priority] to avoid conflicts, but sometimes an environment can only be solved when channel priority is set to `"flexible"`. The catch is that the environment may solve and install but not work! But it's usually worth a try.
+
+This environment is also complex enough that remembering all the channels every time you want to update it is risky. So instead of trying to create it in one line, we might be better off creating it empty, configuring its channel list and priorities, and then installing our software:
+
+```shell-session
+$ mamba create -n psi4_and_bespokefit
+$ conda activate psi4_and_bespokefit
+$ conda config --env --set channel_priority flexible
+$ conda config --env --prepend channels anaconda
+$ conda config --env --prepend channels conda-forge
+$ conda config --env --prepend channels psi4
+$ mamba install openff-bespokefit psi4 ambertools
+```
+
+[environment file]: conda:create-env-file-manually
+[strict channel priority]: conda:concepts-performance-channel-priority
+
+(install_windows)=
+## OpenFF on Windows
+
+OpenFF does not directly support Windows operating systems, though much of our software is pure Python code and may work anyway as long as you can provide our dependencies.
```

Comment:
```suggestion
OpenFF does not support or test on Windows natively. All of our software is pure Python code and may work anyway as long as you can provide our dependencies, but many of these dependencies are not available on Windows and may not be in the foreseeable future.
```

Reply:
Personally prefer emphasizing that we CURRENTLY cannot do anything on Windows because of our upstreams, even though this is a cop-out it shifts the blame away from us and puts a bit of distance between us and the appearance that Windows is not supported because we didn't bother trying.

Reply:
Good call - I love how your phrasing warns that users are gonna have a bad time without feeling like we're saying they're breaking an arbitrary rule.